reference-test/src/test/java/au/org/consumerdatastandards/conformance/ProtectedAPIStepsBase.java

```diff
@@ -30,28 +30,22 @@
 
 public class ProtectedAPIStepsBase extends APIStepsBase {
 
-    public static final String INTROSPECTION_ENDPOINT = "introspection_endpoint";
     public static final String TOKEN_ENDPOINT = "token_endpoint";
 
     private static HashMap<String, String> config;
-    private long lastChecked;
-    private boolean active;
+    private static long lastChecked;
 
     protected RequestSpecification buildHeaders(RequestSpecification given) {
         String accessToken = props.getProperty("access.token");
         String authServer = props.getProperty("auth.server");
         String refreshToken = props.getProperty("refresh.token");
-        if (StringUtils.isNotBlank(authServer) && StringUtils.isNotBlank(refreshToken)) {
+        // Assume the access token, if can be renewed, is good for at least 1 minute
+        if (StringUtils.isNotBlank(authServer) && StringUtils.isNotBlank(refreshToken)
+                && (lastChecked == 0 || System.currentTimeMillis() > lastChecked + 60 * 1000)) {
             try {
                 Map<String, String> config = discoveredInfo(authServer);
-                boolean valid = StringUtils.isNotBlank(accessToken);
-                if (valid) {
-                    valid = checkTokenValidity(accessToken, authServer, config.get(INTROSPECTION_ENDPOINT));
-                }
-                if (!valid) {
-                    accessToken = acquireNewAccessToken(config.get(TOKEN_ENDPOINT));
-                    props.setProperty("access_token", accessToken);
-                }
+                accessToken = acquireNewAccessToken(config.get(TOKEN_ENDPOINT));
+                props.setProperty("access_token", accessToken);
```

Comment:
Clarification: "access_token" or "access.token" here?

Reply:
Should be access.token. Good catch! I'm glad I submitted this PR :)